api/src/indigo_loaders.cpp

```diff
@@ -469,11 +542,14 @@ IndigoObject* IndigoMultilineSmilesLoader::at(int index)
     return next();
 }
 
-CEXPORT int indigoIterateSDF(int reader){INDIGO_BEGIN{IndigoObject& obj = self.getObject(reader);
-
-return self.addObject(new IndigoSdfLoader(IndigoScanner::get(obj)));
-}
-INDIGO_END(-1)
+CEXPORT int indigoIterateSDF(int reader)
+{
+    INDIGO_BEGIN
+    {
+        IndigoObject& obj = self.getObject(reader);
+        return self.addObject(new IndigoSdfLoader(IndigoScanner::get(obj)));
+    }
+    INDIGO_END(-1)

```

Comment:
It turned out that we need to put semicolon after `INDIGO_END()` macro. Refer to issue #267.